linkis-dist/package/db/linkis_ddl.sql

```diff
@@ -106,6 +106,7 @@ CREATE TABLE `linkis_ps_job_history_group_history` (
   `engine_type` varchar(32) DEFAULT NULL COMMENT 'Engine type',
   `execution_code` text DEFAULT NULL COMMENT 'Job origin code or code path',
   `result_location` varchar(500) DEFAULT NULL COMMENT 'File path of the resultsets',
+  `observe_info` varchar(500) DEFAULT NULL COMMENT 'Click the notification information',
```

Comment:
`Click the notification information` -> `the notification information configuration of this job`

Reply:
done